lua/framework/math/quadratures/create_line_quadrature.cc

```diff
@@ -10,7 +10,7 @@
 
 using namespace opensn;
 
-RegisterLuaFunctionAsIs(CreateLineQuadrature);
+RegisterLuaFunctionNamespace(CreateLineQuadrature, aquad, CreateLineQuadrature);
```

Comment:
Is this part of aquad? 
@wdhawkins will this PR complicate your quadrature changes?

Reply:
Line quadrature is used for spatial discretization isn't it? `aquad` should reserved for angular quadratures. Did we discuss anything for spatial quadratures? Should it be just `quad`?

Reply:
Correct, `CreateLineQuadrature` has a "revisit" comment. I suppose that refers to this. There was no discussion regarding `quad` module, IIRC.

Reply:
I would be okay with having a `quad` module or putting the more generic quadratures in `math`.

Reply:
I personally like have a `quad` and `aquad` modules.

Reply:
'aquad' and 'squad'